deploy/custom_resources/custom_authorizer/custom_authorizer_lambda.py

```diff
@@ -16,21 +17,32 @@
 Custom Lambda Authorizer is attached to the API Gateway. Check the deploy/stacks/lambda_api.py for more details on deployment
 """
 
+OPENID_CONFIG_PATH = os.path.join(os.environ.get('custom_auth_url', ''), '.well-known', 'openid-configuration')
+jwt_service = JWTServices(OPENID_CONFIG_PATH)
+
 
 def lambda_handler(incoming_event, context):
     # Get the Token which is sent in the Authorization Header
+    logger.debug(incoming_event)
     auth_token = incoming_event['headers']['Authorization']
     if not auth_token:
-        raise Exception('Unauthorized . Token not found')
+        raise Exception('Unauthorized. Missing identity or access JWT')
 
-    verified_claims = JWTServices.validate_jwt_token(auth_token)
-    logger.debug(verified_claims)
+    # Validate User is Active with Proper Access Token
+    user_info = jwt_service.validate_access_token(auth_token)
+
+    # Validate JWT
+    verified_claims = jwt_service.validate_jwt_token(auth_token[7:])
     if not verified_claims:
```

Comment:
Since the method `validate_jwt_token` is now raising exception this if block can be removed